02_Development/Display/setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
```

Comment:
Lovin' it.  I would also suggest an echo to install docker if it's not already installed.